site/pages/disruption-detail/[disruptionId].page.tsx

```diff
@@ -370,19 +370,19 @@ const DisruptionDetail = ({
             ) : null}
             {duplicateDisruptionPopUpState && csrfToken ? (
                 <Popup
-                    action={`/api/duplicate-disruption${queryParams["template"] ? "?template=true" : ""}`}
+                    action={`/api/duplicate-disruption${disruption.template ? "?template=true" : ""}`}
                     cancelActionHandler={cancelActionHandlerDuplicateDisruption}
                     csrfToken={csrfToken}
                     hiddenInputs={duplicateDisruptionPopUpState.hiddenInputs}
                     continueText="Yes, duplicate"
                     cancelText="No, return"
                     questionText={`Are you sure you wish to duplicate the ${
-                        queryParams["template"] ? "template" : "disruption"
+                        disruption.template ? "template" : "disruption"
                     }?`}
                 />
             ) : null}
             <CsrfForm
-                action={`/api/publish-edit${queryParams["template"] ? "?template=true" : ""}`}
+                action={`/api/publish-edit${disruption.template ? "?template=true" : ""}`}
```

Comment:
Error observed, Template details page -> Add Social Media -> Save without any data -> Back button

Reply:
to do with the query params issue not with my pr bug